exercises/practice/resistor-color-trio/.meta/Example.vb

```diff
@@ -0,0 +1,24 @@
+Imports System
+
+Public Module ResistorColorTrio
+    Private Const KiloOhms As Integer = 1_000
+    Private ReadOnly AllColors As String() = {"black", "brown", "red", "orange", "yellow", "green", "blue", "violet", "grey", "white"}
+    Public Function Label(ByVal colors As String()) As String
+        Dim value = GetValue(colors)
+        Dim unit = GetUnit(value)
+        If unit = "kiloohms" Then value \= 1_000
+        Return String.Format("{0} {1}", value, unit)
+    End Function    
+    End Function
```

Comment:
```suggestion

```

Reply:
This suggested change needs to be accepted.

Reply:
Um, I think the tests were run again without accepting the change to remove the doubled End Function.